plugins/acpi-phat/README.md

```diff
@@ -0,0 +1,20 @@
+Platform Health Assessment Table
```

Comment:
I think you want to actually link back to the ACPI spec (IE here: https://uefi.org/htmlspecs/ACPI_Spec_6_4_html/05_ACPI_Software_Programming_Model/ACPI_Software_Programming_Model.html#platform-health-assessment-table)

Reply:
Fixed.